docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
```

Comment:
This is the docker file which is going to connect to my database. In this case mongodb in atlas